src/hammer-vlsi/defaults.yml

```diff
@@ -842,6 +874,44 @@ sim.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+# Specific inputs for the power tool.
+# These may vary based on the specific tool plugin
+power.inputs:
+  # The path to a place and route database of the design to be analyzed
+  # type: str
+  database: null
+
+  # Enable static power rail analysis
+  # type: bool
+  static_rail: false
```

Comment:
Similar to above. Seems like the more consistent thing to do would be to remove the `static_rail` step if you don't want static rail analysis to be run rather than making this flag false.

Reply:
Yeah will do another pass on the flags.

Reply:
Fixed